vktrace/vktrace_replay/vkreplay_vkreplay.cpp

```diff
@@ -4316,12 +4349,18 @@ void vkReplay::manually_replay_vkUpdateDescriptorSetWithTemplate(packet_vkUpdate
         m_objMapper.remap_descriptorupdatetemplates(pPacket->descriptorUpdateTemplate);
     if (pPacket->descriptorUpdateTemplate != VK_NULL_HANDLE && remappedDescriptorUpdateTemplate == VK_NULL_HANDLE) {
         vktrace_LogError(
-            "Error detected in UpdateDescriptorSetWithTemplate() due to invalid remapped VkDescriptorUpdateTemplateKHR.");
+            "Error detected in UpdateDescriptorSetWithTemplate() due to invalid remapped VkDescriptorUpdateTemplate.");
         return;
     }
 
     // Map handles inside of pData
-    remapHandlesInDescriptorSetWithTemplateData(remappedDescriptorUpdateTemplate, (char *)pPacket->pData);
+    if (VK_NULL_HANDLE != remappedDescriptorUpdateTemplate) {
```

Comment:
Minor point: Could the check for a null handle be moved inside of remapHandlersInDescriptorSetWithTemplateData? The code would be easier to read. Avoiding the function call when it is null is probably not important, since the more common case is the handle is not null.